src/sentry/static/sentry/app/views/settings/components/settingsLayout.jsx

```diff
@@ -27,22 +29,27 @@ class SettingsLayout extends React.Component {
       <React.Fragment>
         <SettingsColumn>
           <SettingsHeader>
-            <Flex align="center" width={1}>
-              <Box flex="1">
-                <SettingsBreadcrumb
-                  params={params}
-                  routes={childRoutes}
-                  route={childRoute}
-                />
-              </Box>
-              <SettingsSearch routes={routes} router={router} params={params} />
-            </Flex>
+            <StyledBreadcrumbWrapper>
+              <SettingsHamburgerIcon
+                priority="link"
+                icon={this.state.navVisible ? 'icon-activity' : 'icon-close'}
+                onClick={() => this.setState({navVisible: !this.state.navVisible})}
```

Comment:
Just a note here, [`setState` is async,](https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous) so it's possible that `this.state.navVisible` is not the most up to date value. When we update state that relies on previous state, it's better practice to use the callback version of `setState()`

```suggestion
                onClick={() => this.setState(state => ({navVisible: !state.navVisible}))}
```